status-page.py

```diff
@@ -7,63 +8,68 @@
 
 class StatusPage:
     def __init__(self):
-        config_data_file = open(sys.argv[1])
-        log_file = open(sys.argv[2])
-        self.config_json = json.load(config_data_file)
-        self.log_json = json.load(log_file)
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            '--config',
+            dest='config_path',
+            help='Path to configuration.json',
+            required=True
+        )
+        parser.add_argument(
+            '--status',
+            dest='api_status_path',
+            help='Path to api_status.json',
+            required=True
+        )
+        args, unknown_args = parser.parse_known_args()
```

Comment:
Is `unknown_args` used anywhere? If not, you should consider using ignoing variable for it.